src/test/scala/is/hail/variant/vsm/VSMSuite.scala

```diff
@@ -155,21 +156,17 @@ class VSMSuite extends SparkSuite {
 
     val s2rdd8 = sc.parallelize(Seq((v1, (s2va1,
       Iterable(Genotype(),
-        Genotype(0),
-        Genotype(2)))),
+        Genotype(0)))),
       (v2, (s2va2,
         Iterable(Genotype(0),
-          Genotype(0),
-          Genotype(1)))))).toOrderedRDD
+          Genotype(0)))))).toOrderedRDD
 
     val s4rdd9 = sc.parallelize(Seq((v1, (s4va1,
       Iterable(Genotype(),
-        Genotype(0),
-        Genotype(2)))),
+        Genotype(0)))),
       (v2, (s4va2,
         Iterable(Genotype(0),
-          Genotype(0),
-          Genotype(1)))))).toOrderedRDD
+          Genotype(0)))))).toOrderedRDD
```

Comment:
I don't understand why we need to remove all these genotypes from the tests. Did some behavior change that I missed?

Reply:
Yes.  The number of genotypes didn't match the number of samples.  Since VSM has a RDD[RegionValue] now, we encode all of these into a region value.  That explicitly creates an array of size nSamples, and if the number of genotypes doesn't match, it crashes.

Reply:
👍